templates/dashboard.yaml

```diff
@@ -32,3 +32,5 @@ spec:
         requests:
           cpu: 500m
           memory: 300Mi
+      image:
```

Comment:
Could we provide some comments here describing the context for overriding the chart's default image?